app/models/news/item.rb

```diff
@@ -31,9 +31,11 @@ def before_validation
     end
 
     def after_save
-      (collection_ids - collections.pluck(:id)).each(&method(:add_collection))
-      (collections.pluck(:id) - (collection_ids & collections.pluck(:id)))
-        .each(&method(:remove_collection))
+      to_add = (collection_ids - collections.pluck(:id))
```

Comment:
nitpicking: we can remove parenthesis here
`to_add = collection_ids - collections.pluck(:id)`

Reply:
lol - normally you're asking me to add the parentheses :smile: I'll get that changed